### DISCUSSION TITLE
Trigger email on failure of flow + flow running on different machine from Server

### DISCUSSION THREAD

**Discussion by ReubenTheDS** [PrefectHQ/prefect#3842] [General]
Hi team/community. First off, great product!! I have two main queries:
1) Is this scenario possible:
On MachinePrefectServer run Prefect's Server
On MachineMyFlow, run a Prefect flow, but have it register to a Prefect agent on MachinePrefectServer.
   
   1.b: Is this the correct config to look at: [Configuration Options](https://docs.prefect.io/orchestration/faq/config.html#connecting-to-a-different-api-endpoint)

2) Whenever a flow fails (on MachineMyFlow), can Prefect's Server on MachinePrefectServer be configured to send out an email to a list of email IDs?
   
   2.b: Can someone please point out the relevant doc.
   2.c: If this isn't exactly possible, is there a workaround?

**Comment by cicdw** [comment]
Hi @ReubenTheDS - currently Prefect Server has no email hooks; however, you can configure [an `EmailTask`](https://docs.prefect.io/api/latest/tasks/notifications.html#emailtask) to run at the end of your flow if any reference tasks fail to trigger an email alert.  Alternatively you could setup an email server and use a webhook cloud hook to achieve your goals, but it would require some work.

FWIW Prefect Cloud does have an off-the-shelf email cloud hook that is available on the free tier.

**Reply by ReubenTheDS** [reply]
Hi @cicdw , thank you for the quick response. I'd already read about EmailTask, but the idea was actually this: if the machine running a Prefect flow goes down, the other machine running Prefect Server would trigger an email
Thanks though. I'll look into what you've suggested.
